Core/GDCore/Events/Parsers/ExpressionParser2.h

```diff
@@ -28,6 +29,8 @@ class ExpressionMetadata;
 
 namespace gd {
 
+using namespace gd::GrammarTerminals;

```

Comment:
Not a good practice to use `using namespace` in a header, but this avoids modifying the entire file now.